assets/app/view/game/tile.rb

```diff
@@ -18,6 +18,7 @@ class Tile < Snabberb::Component
 
       needs :show_coords, default: nil
       needs :show_location_names, default: true
+      needs :show_destinations
```

Comment:
this isn't needed

Reply:
Thanks! I was able to remove the needs from here and in hex. I'm pushing up a commit for this shortly